res/css/views/rooms/_IRCLayout.pcss

```diff
@@ -155,13 +145,43 @@ $irc-line-height: $font-18px;
             }
         }
 
+        .mx_EventTile_line,
+        .mx_EventTile_reply {
+            order: 3;
+            display: flex;
+            flex-direction: column;
+            flex-grow: 1;
+            flex-shrink: 1;
+            min-width: 0;
+        }
+
         .mx_EventTile_reply {
             order: 4;
         }
 
+        .mx_EventTile_msgOption {
+            order: 5;
+        }
+
         .mx_EditMessageComposer_buttons {
             position: relative;
         }
+
+        &.mx_EventTile_info {
+            .mx_ViewSourceEvent, /* For hidden events */
+            .mx_TextualEvent {
+                line-height: $irc-line-height;
+            }
+        }
+
+        /* Suppress highlight thing from the normal Layout. */
+        &:hover.mx_EventTile_verified,
+        &:hover.mx_EventTile_unverified,
+        &:hover.mx_EventTile_unknown {
+            .mx_EventTile_line {
+                border-left: 0;
+            }
+        }
```

Comment:
I checked, and this block seems to have become unnecessary.

Initially it has been added with [this commit](https://github.com/matrix-org/matrix-react-sdk/pull/4531/commits/67249e1e9c17c9886812e1e2e07a71e298d2bb9e) for the implementation of IRC layout on https://github.com/matrix-org/matrix-react-sdk/pull/4531/, but the border declarations for highlighting were removed with [this commit](https://github.com/matrix-org/matrix-react-sdk/commit/3a2aa9d96cea7bedeef51f2cacf89daca3bab434#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919L227-R226) on https://github.com/matrix-org/matrix-react-sdk/pull/7209 to replace `border-left` with `box-shadow`. The current implementation is as below:
https://github.com/matrix-org/matrix-react-sdk/blob/e5f06df3f7ae637f677f8e227039411a63faac9c/res/css/views/rooms/_EventTile.pcss#L248-L261

Since I would not like to add another change on this PR, I would create another PR to remove this block.

Reply:
Created a PR for it: https://github.com/matrix-org/matrix-react-sdk/pull/10451